src/components/layout/v2/text-component-properties.tsx

```diff
@@ -226,6 +256,19 @@ const TextComponentProperties = ({
     </ToggleButton>
   );
 
+  /**
+   * Renders font menu items
+   *
+   * @param font font
+   */
+  const renderFontMenuItem = (font: any) => {
```

Comment:
Could this be typed as string rather than any?

Reply:
Yes definitely. Resolved.